roles/openafs_server/vars/CentOS-8.yaml

```diff
@@ -0,0 +1,55 @@
+---
+# Kerberos commands.
+afs_kadmin: "/usr/bin/kadmin"
+afs_klist: "/usr/bin/klist"
+
+# OpenAFS commands.
+afs_asetkey: "/usr/sbin/asetkey"
+afs_akeyconvert: "/usr/sbin/akeyconvert"
+afs_pts: "/usr/bin/pts"
+afs_vos: "/usr/sbin/vos"
+afs_udebug: "/usr/bin/udebug"
+
+# OpenAFS Installation paths.
+afs_afsconfdir: "/usr/afs/etc"
+afs_afsdbdir: "/usr/afs/db"
+afs_afslocaldir: "/usr/afs/local"
+afs_afslogdir: "/usr/afs/logs"
+afs_afssrvdir: "/usr/afs/bin"
+afs_viceetcdir: "/usr/vice/etc"
+afs_bosconfdir: "/usr/afs/local"
+
+# Service name
+afs_server_service: openafs-server
+
+# OS dependent variables
+afs_firewall: firewalld
+afs_selinux: yes
+afs_selinux_openafs_version: "1.1"
+
+# Note: The bosserver creates the /usr/vice/etc directory on startup if it does not
+# exist in order to create a symlink to the CellServDB for vos and pts. Be sure
+# to set the selinux context before bosserver starts.
+afs_selinux_top_dirs:
+ - "/usr/afs"
+ - "/usr/vice"
+
+# Default selinux mode.
+_afs_selinux_mode: permissive
+
+# Build default options
+_afs_server_configure_opts: >
```

Comment:
I believe this should be _afs_devel_configure_opts and should probably be in /roles/openafs_devel/vars/CentOS-8.yaml



I believe that things get set as follows in roles/openafs_devel/tasks/build.yaml  using the following hierarchy (first match "wins")

For server:
 afs_devel_server_configure_opts       (reserved for user to set)
 afs_devel_configure_opts                   (reserved for user to set)
_afs_devel_configure_opts                  this is reserved for "internal" role use.  But we need to see if a "calling" role (e.g. openafs_server / openafs_client) can override what is set in openafs_devel/vars.

Not sure if your intent is for  "openafs_devel" to have its own default configuration as well as allowing the openafs_server / openafs_client to have their own default configuration.  If so, there may need to be some more hackery in setting this.

Reply:
Thanks

_afs_devel_configure_ops go in openafs_devel. the convention is '_afs_devel" ==> openafs_devel,  _afs_server => openafs_server, afs_client ==> openafs_client, etc.